### DISCUSSION TITLE
I demand the closure of the dissemination of content!

### DISCUSSION THREAD

**Discussion by 9564519** [PHPMailer/PHPMailer#2593] [General]
The fact that the code is open does not mean that it is safe. You are to blame for writing and distributing malware. With his help, intellectual property and data are stolen from the Internet. Don't tell tales how white and fluffy you are.

**Comment by Synchro** [comment]
By your logic, because you breathe the same air as criminals, you must be a criminal. Fortunately most of us are not that stupid. Now go away and enjoy your Christmas.

**Comment by prgcoder** [comment]
Dear Marcus,

Over the years I have watched you have to deal with some very intelligent 
individuals and also some from the other extreme....

Happy Christmas and know that there are people out there that truly 
appreciate your incredible efforts that you go through in supporting this 
wonderful bit of code.

Take care and all the very best to you and your family.

Kind regards,
Andy.

**Reply by Synchro** [reply]
Thank you very much @prgcoder! A merry Christmas to you (and everyone else too)!
